app/locales/mr/intro.json

```diff
@@ -0,0 +1,14 @@
+{
+    "private_kit": "प्राइवेट किट",
+    "intro1_para1": "सुरक्षा आणि गोपनीयता संरक्षणासह डिझाइन केलेले, एमआयटी प्रायव्हेट किट सुरक्षित लोकेशन लॉगिंगची पुढील पिढी आहे.",
+    "next": "पुढे",
+    "back": "मागे",
+    "start": "सुरु करा",
+    "intro2_title1": "100KB पेक्षा कमी",
+    "intro2_para1": "प्राइवेट किट चा ट्रेल जनरेटर आपल्या मोबाईल चा लोकेशन डेटा 100KB मध्ये स्टोअर करतो - एका photo पेक्षा कमी जागा.",
+    "intro2_title2": "आपण प्रभारी आहात",
+    "intro2_para2": "डेटा आपल्या संमतीशिवाय आपला मोबाईल कधीही सोडत नाही.",
+    "intro3_title1": "भविष्य",
```

Comment:
what was the english word that got translated to भविष्य?
because the obvious meaning of भविष्य will be astrology to a layman

Reply:
The English version just says 'The Future' which is a bit cryptic.